examples/policy_definition.txt

```diff
@@ -0,0 +1,30 @@
+## Values used in policy generation
```

Comment:
This file looks like Markdown, perhaps it should be renamed with the extension `.md` (it would make viewing it in GitHub more palatable anyways).

Reply:
Good idea, that can be changed.